doc/api/repl.md

```diff
@@ -78,15 +78,16 @@ The default evaluator supports direct evaluation of JavaScript expressions:
 ```js
 > 1 + 1
 2
-> var m = 2
+> const m = 2
 undefined
 > m + 1
 3
 ```
 
 Unless otherwise scoped within blocks (e.g. `{ ... }`) or functions, variables
-declared either implicitly or using the `var` keyword are declared at the
-`global` scope.
+declared either implicitly or using the `const` or `let` keywords
+are declared at the `global` scope (as well as with the `var` keyword
+outside of functions).
 
```

Comment:
This seems like a cumbersome distinction between `const` and `var`, and `let. Couldn't it just be something like this?

> "variables declared either implicitly, or using the `const`, `let`, or `var` keywords are declared at the `global` scope."

Especially since the phrase is immediately preceded by "Unless otherwise scoped within blocks or functions".